src/lib/tvgPictureImpl.h

```diff
@@ -220,6 +259,24 @@ struct Picture::Impl
         return Result::Success;
     }
 
+    bool mesh(const Polygon* triangles, const uint32_t count, const bool append)
+    {
+        if (!triangles || !count) return false;
+
+        if (append && this->triangles){
+            uint32_t newTriangleCnt = this->triangleCnt + count;
+            this->triangles = (Polygon*)realloc(this->triangles, sizeof(Polygon) * newTriangleCnt);
+            for (int i=0; i<count; i++) this->triangles[this->triangleCnt + i] = triangles[i];
```

Comment:
Could you please add space?
`i<count` -> `i < count`

Reply:
Done